jsdecode/decode_json_test.go

```diff
@@ -229,7 +219,7 @@ var (
 		`
 )
 
-func TesttoComputeInstance(t *testing.T) {
+/*func TesttoComputeInstance(t *testing.T) {
```

Comment:
I'm not sure why half of the content of this file is commented out with this change. Should the change to this file be reverted?